doc/source/user_guide/pyarrow.rst

```diff
@@ -0,0 +1,141 @@
+   df
+
+For PyArrow types that accept parameters, you can pass in a PyArrow type with those parameters
+into :class:`ArrowDtype` to use in the ``dtype`` parameter.
+
+.. ipython:: python
+
+   import pyarrow as pa
+   list_str_type = pa.list_(pa.string())
+   ser = pd.Series([["hello"], ["there"]], dtype=pd.ArrowDtype(list_str_type))
+   ser
+
+   from datetime import time
+   idx = pd.Index([time(12, 30), None], dtype=pd.ArrowDtype(pa.time64("us")))
+   idx
+
+   from decimal import Decimal
+   decimal_type = pd.ArrowDtype(pa.decimal128(3, scale=2))
+   data = [[Decimal("3.19"), None], [None, Decimal("-1.23")]]
+   df = pd.DataFrame(data, dtype=decimal_type)
+   df
+
+If you already have an :external+pyarrow:py:class:`pyarrow.Array` or :external+pyarrow:py:class:`pyarrow.ChunkedArray`,
+you can pass it into :class:`.arrays.ArrowExtensionArray` to construct the associated :class:`Series`, :class:`Index:`
+or :class:`DataFrame` object.
+
+.. ipython:: python
+
+   pa_array = pa.array([{"1": "2"}, {"10": "20"}, None])
+   ser = pd.Series(pd.arrays.ArrowExtensionArray(pa_array))
+   ser
+
+
+Operations
+----------
+
+PyArrow data structure integration is implemented through pandas' :class:`~pandas.api.extensions.ExtensionArray` :ref:`interface <extending.extension-types>`;
+therefore, supported functionality exists where this interface is integrated within the pandas API. Additionally, this functionality
+is accelerated with PyArrow `compute functions <https://arrow.apache.org/docs/python/api/compute.html>`__ where available. This includes:
+
+* Numeric aggregations
+* Numeric arithmetic
+* Numeric rounding
+* Logical and comparison functions
+* String functionality
+* Datetime functionality
+
+.. ipython:: python
+
+   ser = pd.Series([-1.545, 0.211, None], dtype="float32[pyarrow]")
+   ser.mean()
+   ser + ser
+   ser > (ser + 1)
+
+   ser.dropna()
+   ser.isna()
+   ser.fillna(0)
+
+   ser_str = pd.Series(["a", "b", None], dtype="string[pyarrow]")
+   ser_str.str.startswith("a")
+
+   from datetime import datetime
+   pa_type = pd.ArrowDtype(pa.timestamp("ns"))
+   ser_dt = pd.Series([datetime(2022, 1, 1), None], dtype=pa_type)
+   ser_dt.dt.strftime("%Y-%m")
+
+I/O Reading
+-----------
+
+PyArrow also provides IO reading functionality that has been integrated into several pandas IO readers. The following
+functions provide an ``engine`` keyword that can dispatch to PyArrow to accelerate reading from an IO source.
+
+* :func:`read_csv`
+* :func:`read_json`
+* :func:`read_orc`
+* :func:`read_feather`
+
+.. ipython:: python
+
+   import io
+   data = io.StringIO("""a,b,c
+      1,2.5,True
+      3,4.5,False
+   """)
+   df = pd.read_csv(data, engine="pyarrow")
+   df
+
+By default, these functions, and all other IO reader functions, return NumPy-backed data. These readers can return
+PyArrow-backed data by specifying ``use_nullable_dtypes`` with the global configuration option ``"mode.dtype_backend"``
```

Comment:
Could you reference the global option to set ``use_nullable_dtypes``as well?

Reply:
I find this sentence a bit confusing, seems like we're first suggesting to use `mode.dtype_backend='use_nullable_dtypes'`. Maybe we can write it like `by specifying the parameter use nullable_dtypes *and* the global configuration ...`, or something else that helps understand this better.